packages/template-retail-react-app/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@salesforce/retail-react-app",
-  "version": "2.2.0-preview.0",
+  "version": "2.2.0-preview.2",
```

Comment:
This preview version skips `1` and  jumps straight to `2`.

Reply:
good catch, it is fixed with `1`